vm/safepoints.cpp

```diff
@@ -35,11 +35,18 @@ void factor_vm::handle_safepoint(cell pc) {
   faulting_p = false;
 
   if (atomic::load(&safepoint_fep_p)) {
+    atomic::store(&safepoint_fep_p, false);
```

Comment:
@mrjbq7: "is there a reason you can't just enqueue_fep() and then t to throw an exception into Factor?"

The main use case for the entire feature is when we have no console to `t` in. Otherwise we could just use Ctrl-C in the first place.